3-cloud-computing/3.2.3-auto-scaling.md

```diff
@@ -16,21 +16,20 @@ Read more about User Data in the [AWS documentation](https://docs.aws.amazon.com
 
 ## Exercise 1 - Create EC2 instance with User Data
 
-In this exercise we will create an EC2 instance and use a User Data script to provision the instance as host for a Java web application.
+In this exercise we will create an EC2 instance and use a User Data script to provision the instance as host for a Java web application. We recommend configuring a machine to run [Spring's PetClinic Project](https://github.com/spring-projects/spring-petclinic) on [CentOS Stream 9](https://www.centos.org/download/aws-images/). But you are free to do this exercise with another web application and distro.
```

Comment:
Love these changes.
- would have been so helpful to know when I went threw.
- centos stream 9 👍